schedule/virt_autotest/vgpu_tests.yaml

```diff
@@ -0,0 +1,17 @@
+name: vgpu_tests.yaml
+description:    >
+    Maintainer: qe-virt@suse.de, jcao@suse.com
+    vGPU short and full passthrough tests on UEFI guests without secure boot
+schedule:
+    - autoyast/prepare_profile
+    - installation/ipxe_install
+    - autoyast/installation
+    - virt_autotest/login_console
+    - '{{install_guest}}'
+    - virt_autotest/vgpu
+conditional_schedule:
+  install_guest:
+    VGPU_TEST:
+      full:
```

Comment:
Is it correct understanding that full has vm installation while no vm installation for short? Then for short case, where the vm would come from?

Reply:
As we talked, the short test will only install the vgpu manager and switch on its sriov ability on host. There is no guests involving in this test, so we can skip installing guests.

Reply:
Thanks for the explanation again. Now I recall it.